package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node index.js",
     "db-migrate": "dropdb bitballs --if-exists && createdb bitballs && ./node_modules/db-migrate/bin/db-migrate up",
-    "install": "cd public/ && npm install"
+    "test": "cd public/ && npm test"
```

Comment:
Yes--the issue you're encountering locally where this isn't working is messing up the build on travis.